include/dlaf/factorization/factorization.h

```diff
@@ -9,11 +9,4 @@
 //
 #pragma once
 
-#include "dlaf/types.h"
-
-namespace dlaf {
-
-template <Backend execution>
-struct Solver;
-
-}
+#include "dlaf/factorization/cholesky.h"
```

Comment:
This header should probably replace the `factorization.h` header outside of the `factorization/` directory. That header has a wrong include statement in it (dlaf/factorization/mc.h), this one has the right include statement.